docs/python/debugging.md

```diff
@@ -47,11 +47,11 @@ The details of configuration properties are covered later in this article under
 
 ## Additional configurations
 
-By default, VS Code shows only the most common configurations provided by the Python extension. You can select other configurations to include in `launch.json` by using the **Add Configuration** command shown in the list and the `launch.json` editor. When you use the command, VS Code prompts you with a list of all available configurations (be sure to scroll down to see  all the Python options):
+By default, VS Code shows only the most common configurations provided by the Python extension. You can select other configurations to include in `launch.json` by using the **Add Configuration** command shown in the list and the `launch.json` editor. When you use the command, VS Code prompts you with a list of all available configurations (be sure to click on the "Python" option):
```

Comment:
"select" is more inclusive language than "click" (reader might not be using a mouse).
Also the convention is to use bold for UI text.
So propose: "(be sure to select the **Python** option)"

Reply:
Great point, thank you!